### PR TITLE
PLANET-6988: New layout applied to tag links

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
Ref: https://jira.greenpeace.org/browse/PLANET-6988

## Description
New layout applied to Tag links as a part of the new identity project.

## Testing
Navigate any page with tags, for example the [forest category page](https://www-dev.greenpeace.org/test-nix/tag/forests/).

Remember to enable the new layout from the appearance section

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [nix](https://www-dev.greenpeace.org/test-nix) | [admin](https://www-dev.greenpeace.org/test-nix/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-nix/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-nix) | [composer-local.json](https://github.com/greenpeace/planet4-test-nix/blob/main/composer-local.json)

:watch: 2023.03.17 21:43:05

File: assets/src/scss/new-identity/style.scss [hunk 1]
```diff
@@ -9,4 +9,22 @@
...
+  --post-tag-button--hover--color: var(--grey-900);
+  --post-tag-button--background-color: var(--gp-green-100);
+  --post-tag-button--hover--background-color: var(--gp-green-200);
+  --post-tag-button--active--background-color: var(--gp-green-200);
+  --post-tag-button--visited--color: var(--grey-900);
+  --post-tag-button--active--color: var(--grey-900);
+}
+
+.post-tag-button {
+  color: var(--grey-900) !important;
```
Comment (mleray) [inline assets/src/scss/new-identity/style.scss:25]: We do have a `--post-tag-button--color` variable, why aren't you using it instead of this line? 🤔

File: assets/src/scss/new-identity/style.scss [hunk 2]
```diff
@@ -9,4 +9,22 @@
   --block-columns--step-number--background: var(--gp-green-100);
   --block-columns--step-number--color: var(--grey-900);
   --block-columns--card-header--background: var(--p4-dark-green-800);
+  --link--focus--outline: 2px solid #99c3ff;
+  --top-page-tags--tag-item--background: var(--gp-green-100);
```
Comment (mleray) [inline assets/src/scss/new-identity/style.scss:13]: This one feels unrelated?

File: assets/src/scss/new-identity/style.scss [hunk 3]
```diff
@@ -9,4 +9,22 @@
   --block-columns--step-number--background: var(--gp-green-100);
   --block-columns--step-number--color: var(--grey-900);
   --block-columns--card-header--background: var(--p4-dark-green-800);
+  --link--focus--outline: 2px solid #99c3ff;
```
Comment (mleray) [inline assets/src/scss/new-identity/style.scss:12]: This is going to affect all links, not only the post tags, was that properly tested?

File: assets/src/scss/new-identity/style.scss [hunk 4]
```diff
@@ -9,4 +9,19 @@
   --block-columns--step-number--background: var(--gp-green-100);
   --block-columns--step-number--color: var(--grey-900);
   --block-columns--card-header--background: var(--p4-dark-green-800);
+  --button--focus--outline: 2px solid #99c3ff;
+  --button--focus--outline-offset: 2px;
+  --top-page-tags--tag-item--background: var(--gp-green-100);
+  --post-tag-button--font-weight: 700;
+  --post-tag-button--border-color: var(--gp-green-100);
+  --post-tag-button--hover--color: var(--grey-900);
```
Comment (mleray) [inline assets/src/scss/new-identity/style.scss:17]: The border color should also be changed on hover, right now it looks a bit weird

File: assets/src/scss/new-identity/style.scss (hunk 3 above)
Comment (GP-Dan-Tovbein) [reply assets/src/scss/new-identity/style.scss]: This was part of the previous implementation. I forgot to remove it 🙏

File: assets/src/scss/new-identity/style.scss (hunk 2 above)
Comment (GP-Dan-Tovbein) [reply assets/src/scss/new-identity/style.scss:14]: This was part of the previous implementation. I forgot to remove it 🙏

File: assets/src/scss/new-identity/style.scss [hunk 5]
```diff
@@ -9,4 +9,19 @@
...
+  --post-tag-button--background-color: var(--gp-green-100);
+  --post-tag-button--hover--background-color: var(--gp-green-200);
```
[the remark this answers is not in the document]
Comment (GP-Dan-Tovbein) [reply assets/src/scss/new-identity/style.scss:21]: You're correct! I've just updated the hover layout.

File: assets/src/scss/new-identity/style.scss [hunk 6]
```diff
@@ -15,4 +15,23 @@
...
+  --post-tag-button--background-color: var(--gp-green-100);
+  --post-tag-button--border-color: var(--gp-green-100);
+  --post-tag-button--color: var(--grey-900);
+  --post-tag-button--hover--background-color: var(--gp-green-200);
+  --post-tag-button--hover--border-color: var(--gp-green-200);;
+  --post-tag-button--hover--color: var(--grey-900);
+  --post-tag-button--visited--color: var(--grey-900);
+}
+
+.post-tag-button {
```
Comment (mleray) [inline assets/src/scss/new-identity/style.scss:33]: Maybe instead of defining this here, you could just add the hover border variable where we have the rest of the `post-tag-button` styles (so [here](https://github.com/greenpeace/planet4-master-theme/blob/af4d9553c7febd7d15a0403271acf26744f18f07/assets/src/scss/components/_buttons.scss#L285-L288)) and set it to `var(--link--color)`?

Comment (mleray) [reply assets/src/scss/new-identity/style.scss:33]: No, it won't 🤔 I'm not saying you should change the `--link--color` variable, sorry if that was confusing. I'm just saying you could directly write in `_buttons.scss`:
```
&:hover {
      background-color: var(--link--color);
      border-color: var(--link--color);
      color: $white;
}
```
and that would create the `--post-tag-button--hover--border-color` that we can override here! Does it make sense?

Comment (GP-Dan-Tovbein) [reply assets/src/scss/new-identity/style.scss]: Yes, that really make sense!

File: assets/src/scss/new-identity/style.scss (hunk 2 above)
Comment (mleray) [reply assets/src/scss/new-identity/style.scss:20]: You marked this as resolved, but the `--top-page-tags--tag-item--background` override is still here 🤔

Comment (GP-Dan-Tovbein) [reply assets/src/scss/new-identity/style.scss]: Sorry, I forgot to push it 🤦

File: assets/src/scss/new-identity/style.scss [hunk 7]
```diff
@@ -15,4 +15,16 @@
   --button-primary--color: var(--grey-900);
   --button-primary--visited--color: var(--grey-900);
   --button-primary--hover--color: var(--grey-900);
+  --button--focus--outline: 2px solid #99c3ff;
```
Comment (mleray) [inline assets/src/scss/new-identity/style.scss:18]: This will impact all buttons, right? Not just post tags?

Comment (GP-Dan-Tovbein) [reply assets/src/scss/new-identity/style.scss:18]: Exactly! It's part of the [new layout for buttons](https://www.figma.com/file/9NtbY8n3at8uOEJTsLrETb/P4-Design-System?node-id=370-35690&t=whzv8fJGAqVBOf1L-4)

Comment (mleray) [reply assets/src/scss/new-identity/style.scss:18]: Do we actually need to override it? It seems it already looks as expected in the current buttons and post tags

Comment (mleray) [reply assets/src/scss/new-identity/style.scss:18]: Yeah I think we can directly change it in the typography file, since it should be applied regardless of the new identity! 🙂

File: assets/src/scss/new-identity/style.scss [hunk 8]
```diff
@@ -15,4 +15,15 @@
   --button-primary--color: var(--grey-900);
   --button-primary--visited--color: var(--grey-900);
   --button-primary--hover--color: var(--grey-900);
+  --button--focus--outline-offset: 2px;
```
Comment (mleray) [inline assets/src/scss/new-identity/style.scss:18]: This can be removed, since it's the same as the current value 🙂

**Review by mleray — APPROVED** [review]
Looks good to me 👏
